modules/bigbio/thermorawfileparser/main.nf

```diff
@@ -0,0 +1,64 @@
+process THERMORAWFILEPARSER {
+    tag "$meta.mzml_id"
+    label 'process_low'
+    label 'process_single'
+    label 'error_retry'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/thermorawfileparser:1.4.5--h05cac1d_1' :
+        'biocontainers/thermorawfileparser:1.4.5--h05cac1d_1' }"
+
+    stageInMode {
+        if (task.attempt == 1) {
+            if (task.executor == "awsbatch") {
+                'symlink'
+            } else {
+                'link'
+            }
+        } else if (task.attempt == 2) {
+            if (task.executor == "awsbatch") {
+                'copy'
+            } else {
+                'symlink'
+            }
+        } else {
+            'copy'
+        }
+    }
+
+    input:
+    tuple val(meta), path(rawfile)
+
+    output:
+    tuple val(meta), path("*.mzML"), emit: mzmls_converted
```

Comment:
Could the output format be set to dynamic? As this could be mzML, MGF Parquet or others.

Reply:
Yes, next commit is for that, we will set the output by arguments. This is needed because we have a new workflow that will need mgf.